src/book/02-projects/01-nubots/03-build-system.mdx

```diff
@@ -312,14 +312,23 @@ config/
 └── ExampleModule.yaml
 ```
 
-The file `config/ExampleMoldule.yaml` must exit for every module. This file defines all of the configuration for the module. `config/nugus1/ExampleModule.yaml` is an optional file and allows you to provide robot-specific overrides for specific configuration values, in this case it would provide overrides for the `nugus1` robot. Similarly, `config/robocup/ExampleModule.yaml` allows for the `robocup` role to further override configuration values.
+The file `config/ExampleMoldule.yaml` must exit for every module. This file defines all of the configuration for the module.
```

Comment:
Doesn't have to exist, there are quite a few modules that don't have configuration at all

Reply:
Also "exit" => "exist"